src/core/modules/give-away/give-away.service.js

```diff
@@ -11,7 +11,8 @@ class GiveAwayServiceImpl {
             return failResponse('Failed', 'Invalid command! Time cannot be set to the past & Quantity must be a positive number');
         }
         ScheduleService.scheduleJob(channelId, messageId, data, DiscordService.client);
-        return successResponse('Successfully setup give-away event', `[${data.message} - Time to roll: ${data.date.toLocaleString()}]`);
+        return successResponse('Successfully setup give-away event',
+            `[${data.message} - Time to roll: ${data.date.date()}/${data.date.month()}/${data.date.year()} - ${data.date.hour()}:${data.date.minute()}:00}]`);
```

Comment:
You should define a function ( put it in util ) to get date with that format, so as you can reuse it.

Reply:
okay I got it

Reply:
I've just forced push